### PR TITLE
Update fusion backend version

### PR DESCRIPTION
**Pull request by LukasZahradnik**
##### SUMMARY

This PR updates the version format of the backend to more generic "v1".

##### COMPONENT NAME

`module_utils/fusion.py`

### CODE REVIEW

File: plugins/module_utils/fusion.py [hunk 1]
```diff
@@ -61,7 +61,7 @@
 ENV_APP_ID = "FUSION_APP_ID"  # replaced by ENV_ISSUER_ID
 ENV_HOST = "FUSION_HOST"  # replaced by ENV_API_HOST
 DEP_VER = "2.0.0"
-BASE_PATH = "/api/1.1"
+BASE_PATH = "/api/v1"
```
Comment (dimleevik) [inline plugins/module_utils/fusion.py:64]: I am not sure if fusion has released this v1 path feature. Maybe lets remain 1.1 ?

Comment (LukasZahradnik) [reply plugins/module_utils/fusion.py:64]: Right. Changing the PR to draft

Comment (andreepyro) [reply plugins/module_utils/fusion.py:64]: Now API 1.2 was release.

Comment (dimleevik) [reply plugins/module_utils/fusion.py:64]: Moved this PR from draft to ready

**Review by dimleevik — APPROVED** [review]

**Comment by andreepyro** [comment]
@dimleevik Isn't the endpoint `1.2` now? https://github.com/PureStorage-OpenConnect/fusion-python-sdk/blob/1b801fc4da76ebdf591df222d1cb2e61238fe688/fusion/configuration.py#L49C9-L49C9

**Comment by LukasZahradnik** [comment]
@andreepyro `/v1` is an alias for `/1.x`. So it shouldn't matter

**Comment by andreepyro** [comment]
@dimleevik you're right I thought the change is from `1` to `1.1`

However, I believe we depend on Fusion API `1.2` release, right? Then we should change the ansible prerequisite in `plugins/module_utils/prerequisites.py` file to `("fusion", "purefusion", ">=1.2.0,<2.0"),`

**Comment by LukasZahradnik** [comment]
@andreepyro Right. I think we should somehow automate this instead of looking for hidden versions

**Comment by dimleevik** [comment]
@andreepyro, @LukasZahradnik AFAIK /v1 is alias for /latest which is alias to /v1.2

**Comment by dimleevik** [comment]
@andreepyro, @LukasZahradnik Actually for this change we don't need to change python version, this change works with all versions, since we only provide hostname to sdk, so it will send requests to server with /v1 suffix

**Pull request closed without merging**